### ISSUE TITLE
[EN][NPC][Wintergasp] Goblin mechanic

### ISSUE THREAD

**Issue by Digglewiggle** [Atlantiss/BugTracker#7429]
Hi. I noticed that NPC who is giving an option for us to build a vehicle in workshop on Wintergasp, world battleground is placed in the ground and we are unable to build a vehicle.
http://www.wowhead.com/npc=30400/goblin-mechanic

**Comment by Llagas** [comment]
Yes, and its only the alliance version of the npc that is missing, so it is impossible for alliance to win wintergrasp.

**Comment by rdoherty0** [comment]
And now, it is therefore impossible for Alliance to finish the Lunar Event, since a necessary Elder is inside the Wintergrasp fort.
